### ISSUE TITLE
New Line Character does not get converted to a <br>

### ISSUE THREAD

**Issue by Pfenya** [freb97/nuxt-strapi-blocks-renderer#2]
Hi,

first off: thank you for the work on this!

I am currently running into a case where I need to break some text in Strapi for the visual representation - but when I add a new line in Strapi (Shift Enter), it makes it to the API response but the renderer takes it out of the code. Is there an easy way to make this work?

relevant part of the API response from Strapi
`{"id":56,"title":"Hände nass machen.","text":[{"type":"paragraph","children":[{"text":"Jetzt heißt es „Wasser marsch“ \nund die Hände nass machen","type":"text"}]}]},`

Nuxt code
```
        <div v-if="solution.steps" v-for="(step, i) in solution.steps"  :class="`pipe-text pipe-text-${i+1}`" >
            <p>{{step.title}}</p>
            <StrapiBlocksText :nodes="step.text" />
        </div>
```

HTML output:
```<div class="pipe-text pipe-text-6" data-v-inspector="components/Info/SurroundSolution.vue:22:9" data-v-04af98a3="">
<p data-v-inspector="components/Info/SurroundSolution.vue:23:13" data-v-04af98a3="">Hände nass machen.</p>
<p data-v-inspector="node_modules/nuxt-strapi-blocks-renderer/dist/runtime/components/blocks/ParagraphNode.vue:2:5"
 data-v-04af98a3="">Jetzt heißt es „Wasser marsch“ 
und die Hände nass machen</p>
</div>
```

The HTML preserves the line break but the browser then does not since it will get rid of arbitrary line breaks in elements.

I also tried with `&shy;` but that does simply get printed to the frontend and not interpreted as a html character.

Any help in pointing me in the right direction would be greatly appreciated!

**Comment by freb97** [comment]
Hey @Pfenya,

Thanks so much for the feedback! I am at work right now but i will look into this as soon as i find the time! Will keep you posted.

**Comment by freb97** [comment]
@Pfenya I just released version 1.0.6 with some changes to the text handling of the nodes, so line breaks should work now.

If you run into any more problems please let me know! :)

**Comment by Pfenya** [comment]
Thank you so much for the quick response! I will give it a try
